pjsip/src/pjsip/sip_transport.c

```diff
@@ -1653,6 +1653,11 @@ PJ_DEF(pj_ssize_t) pjsip_tpmgr_receive_packet( pjsip_tpmgr *mgr,
 		    mgr->on_rx_msg(mgr->endpt, PJSIP_ERXOVERFLOW, rdata);
 		    /* Exhaust all data. */
 		    return rdata->pkt_info.len;
+     		} else if (msg_status == PJSIP_EMISSINGHDR) {
+        	    /* No content-length header which is not allowed. */
```

Comment:
Actually pjsip_find_msg doesn't parse the entire message it only looks for the content length header to check if we have received the entire message.  So I think we can be explicit in our log.